wathigo/iReporter2
# Ft delete endpoint #162339837

**What his PR do?**
Adds a delete request endpoint to the views module
**Description of the task to be completed**
Create a delete endpoint in the views module such that when a a delete request is sent, the response should be either an empty data structure or the updated content with the specified id removed
**How can this be manually tested**
git clone https://github.com/wathigo/iReporter2.git to you local machine
navigate to app/api/v1 and open views module to see the implementation of the endpoint
